bayesian/api_v1.py

```diff
@@ -391,6 +384,10 @@ def get(external_request_id):
                         "overall_score": 0,
                         "magnitude": 0
                     }
+                
+                # Adding topics from the recommendations
+                stack_recommendation = get_item_from_list_by_key_value(recommendations, "manifest_file_path", stack.get("manifest_file_path"))
```

Comment:
where is `get_item_from_list_by_key_value` implemented? Have you missed adding any files?

Reply:
this method is already present in utils.py